platforms/core-configuration/kotlin-dsl-integ-tests/src/integTest/kotlin/org/gradle/kotlin/dsl/integration/KotlinDslJvmTargetIntegrationTest.kt

```diff
@@ -148,7 +148,6 @@ class KotlinDslJvmTargetIntegrationTest : AbstractKotlinIntegrationTest() {
         withFile("plugin/src/main/kotlin/some.gradle.kts", printScriptJavaClassFileMajorVersion)
 
         gradleExecuterFor(arrayOf("check", "publish"), rootDir = file("plugin"))
-            .withJavaHome(currentJvm.javaHome)
```

Comment:
🤔 why do we no longer pass the `currentJvm` along?

Reply:
Added this back. 

By default the executor adopts the current JVM. For the first call this is not necessary, but for subsequent calls it is, as we set it to the newerJvm in-between.